apps/dashboard-service/src/dashboard/total-section-header.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Get, Query, Controller } from "@nestjs/common";
+import { ApiOperation } from "@nestjs/swagger";
+import { DashboardQueryDto } from "./dto/dashboard-query.dto";
+import { JsonApiResponse } from "@terramatch-microservices/common/decorators";
+import { DelayedJobDto } from "./dto/delayed-job.dto";
+import { CacheService } from "./dto/cache.service";
+import { DelayedJob } from "@terramatch-microservices/database/entities";
+
+@Controller("v3/dashboard/totalSectionHeader")
+export class TotalSectionHeaderController {
+  constructor(private readonly cacheService: CacheService) {}
+
+  @Get()
+  @JsonApiResponse(DelayedJobDto)
+  @ApiOperation({ summary: "Get total section header" })
+  async getTotalSectionHeader(@Query() query: DashboardQueryDto) {
+    const cacheKey = `dashboard:total-section-header|${this.cacheService.getCacheKeyFromQuery(query)}`;
+    const cachedData = await this.cacheService.get(cacheKey);
+    if (cachedData == null) {
+      const delayedJob = await DelayedJob.create({});
+      await this.cacheService.getTotalSectionHeader(cacheKey, query, delayedJob.id);
+      const delayedJobDto = new DelayedJobDto(delayedJob);
+      return delayedJobDto;
+    }
+    return JSON.parse(cachedData);
```

Comment:
Every v3 controller should be returning a JSON API structure (typically by using the `buildJsonApi()` builder). Also, rather than returning the delayed job DTO, it should return the delayed job ID, and let the FE delayed job connection / middleware (which we'll need to add an integration for in v3) handle the polling of the delayed job, similar to how it's done in v2.